src/index.html

```diff
@@ -13,7 +17,11 @@ <h1 class="task-valve">Task Table</h1>
 
         <div class="top-block">
             <h2>Today's To Do</h2>
+            <form class="form">
             <input class="add-input" type="text" placeholder="Add to your list..." required>
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to fix this bug in the UI so it looks nicer

![Screenshot_20230616_175916](https://github.com/maths29/Todo-List/assets/25227022/d16c625f-4179-4f13-916b-ee8d57a4afc8)